docs/start/index.rst

```diff
@@ -81,6 +89,25 @@ If you have installed with git, pull the QCoDeS repository using your
 favourite method (git bash, git shell, github desktop, ...). There are
 new commits to the repository daily.
 
+Keeping your environment up to date
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Additional dependencies are periodically added to the QCoDeS environment and
+new versions of packages that QCoDeS depends on are released.
+
+To keep the QCoDeS environment upto date you can run
+
+.. code:: bash
+
+   conda env update
+
+from the root of your QCoDeS git repository. Alternatively you can grab a new
+version of the ``environment.yml`` file from above and run the same command from a
+directory containing that file. This will update all the QCoDeS dependencies to the
+latest recommended version. Note that if you install packages yourself into the same
```

Comment:
Shall here be a "new paragraph" in order to separate the "Note .." from the above text?

Reply:
I only pined it originally because there was a abi version mismatch between the versions of sip and pyqt on pypi. (sip is a direct dependency of pyqt5 to generate the c++ wrappers and only used for that) so in principle we should not need to pin it at all. I don't expect that will ever happen for the conda package